aptos-move/aptos-workspace-server/src/main.rs

```diff
@@ -169,7 +231,7 @@ async fn main() -> Result<()> {
 
     println!("Test directory: {}", test_dir.path().display());
 
-    spawn_node(test_dir.path()).await?;
+    start_all_services(test_dir.path()).await?;
```

Comment:
Instead of just starting the services and then looping, we should be awaiting all of their tokio tasks / futures. Otherwise any one of these services could fail silently.